crux-bench/src/crux/bench/watdiv_crux.clj

```diff
@@ -75,12 +75,7 @@
                                (bench/run-bench (format "query-%d" idx)
                                                 {:result-count (count (crux/q (crux/db node) (sparql/sparql->datalog q)))}))))))))))]
 
-    (-> [(first watdiv-results)
-         (->query-result (rest watdiv-results))]
```

Comment:
this returned the raw results so that the raw results were put into CloudWatch, the summarised results to Slack - let's preserve this behaviour